inginious/frontend/plugins/custom_input/static/custom_input.js

```diff
@@ -0,0 +1,76 @@
+function getTaskIdFromUrl() {
+    var urlTokens = window.location.pathname.split("/");
```

Comment:
I encourage you to use `let` instead of `var`. When you declare a variable with `var` it is going to be 'visible' in the whole code after its declaration and with `let`, the variable is only visible in the scope it was declared. 
It may avoid some problems related to variable naming.